css/main.css

```diff
@@ -325,3 +325,90 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.footer {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer .footer-header {
+  padding: 20px;
+  height: 216px;
+  margin: 90px 24px 80px;
+}
+
+.footer .footer-form {
+  margin: 20px 5px;
+}
+
+.footer .footer-header p {
+  text-align: center;
+  color: #172b4d;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  font-weight: bolder;
+  line-height: 42px;
+}
+
+.footer form {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer .footer-form input {
+  margin: 10px 5px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 4px;
+  width: 90%;
+  height: 48px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  align-items: center;
+  color: #979493;
+  padding: 5px;
+  outline: none;
+  border: none;
+  border-bottom: 1px solid #d0d9d4;
+}
+
+.footer .footer-form textarea {
+  margin: 10px 5px;
+  background: #fbf8f7;
+  color: #3c3a39;
+  border: none;
+  outline: none;
+  border-radius: 4px;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  width: 65%;
+  box-sizing: border-box;
+  padding: 12px;
+  font-family: "Inter", sans-serif;
+  height: 114px;
+}
+
```

Comment:
* [ ] Great job on following the design from [Figma](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=48%3A838), but there are still other minor things that could improve your design Please, refer to the images below for some improvement(s).


> || **YOUR DESIGN** || FIGMA DESIGN ||
> |-|---------|-|---------|-|
> ||  ![width_2](https://user-images.githubusercontent.com/59397159/163323014-264d0c65-90f9-4dca-9969-b4f86aaa0f3a.png) || ![width](https://user-images.githubusercontent.com/59397159/163323135-c59e35b1-f732-441a-bb0c-4c1f65e8cfd6.png) || 

- As you can see your `textarea's width` is too small. You can play with it to fix the issue